<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryPage</class>
 <widget class="QWidget" name="LibraryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>library page</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="listPage"/>
     <widget class="CategoryPage" name="detailPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CategoryPage</class>
   <extends>QWidget</extends>
   <header>categorypage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
